src/TagsInput/components/TagsContainer.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import * as React from 'react';
+import { View, ScrollView } from 'react-native';
+import { Badge } from '../../Badge';
+import { StyleSheet } from '../../PlatformStyleSheet';
+import { MIN_WIDTH } from '../constants';
+
+type Props = {|
+  +tags?: string[],
+  +fontSize: number,
+  +maxWidth: string | number,
+|};
+
+class TagsContainer extends React.PureComponent<Props> {
+  static defaultProps = {
+    maxWidth: 'auto',
+  };
+
+  render() {
+    const { tags, fontSize, maxWidth } = this.props;
+    if (!tags || tags.length < 1) {
+      return null;
+    }
+
+    return (
+      <View>
+        <ScrollView
+          centerContent
+          horizontal
+          style={[styles.container, styles.row, { maxWidth }]}
+          showsHorizontalScrollIndicator={false}
+          contentContainerStyle={{
+            alignItems: 'center',
```

Comment:
Could you extract the style?

Reply:
I tried before but I got back this flow error,
<img width="935" alt="screen shot 2019-01-11 at 16 01 13" src="https://user-images.githubusercontent.com/6354326/51041416-2e83ca00-15ba-11e9-9ddb-e0440c2eb098.png">

Reply:
Then we might need to address how our `StyleSheet.create` works, compared to RN's native one. Or use `StyleSheet.create` from RN implementation.
@tbergq What do you think about this issue?

Reply:
OK, I played with the flow for while but without success so I decided to use StyleSheet.create from RN implementation.